taskboot/cli.py

```diff
@@ -337,6 +337,11 @@ def main() -> None:
     cargo_publish_cmd = commands.add_parser(
         "cargo-publish", help="Publish a crate on crates.io"
     )
+    cargo_publish_cmd.add_argument(
+        "--do-not-fail",
```

Comment:
It's a bit too generic :P
Maybe `--ignore-published`?

Reply:
Too generic without the `help` description :P 
Anyway, I can change that

Reply:
It is still too generic, what if we add another option to ignore some error? We call it `--do-not-fail2`? :D

Reply:
The context would be different, but ok, let's change that